app/src/main/java/com/trainstation/frettirtest2/MyViewPagerFragment.java

```diff
@@ -8,36 +9,27 @@
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.ListView;
-
+import android.widget.Toast;
 import com.trainstation.frettirtest2.responses.NewsResponse;
 import com.trainstation.frettirtest2.responses.NewsResultObject;
+import rx.Observer;
 
 import java.util.ArrayList;
 import java.util.List;
 
-import rx.Observer;
-
 /**
  * Created by Helgi on 17/01/16.
  */
-public class MyViewPagerFragment extends Fragment{
+public class MyViewPagerFragment extends Fragment
+{
+    private static final String LOGTAG = MyViewPagerFragment.class.getSimpleName();
```

Comment:
when Logging something out, e.g. here: https://github.com/helgio/frettaApp/pull/2/files#diff-bd250610c6b3fa979c8e0f1b71926f40R49, you always need to set the tag or logtag (the first parameter in the `Log.e()` method), which is the keyword you see in the logcat, when the app is logging out informations,

so you may ask "what are you doing here?" --> I am declaring the logtag in one place for the whole class, instead of doing it when logging out (like we were doing here: https://github.com/helgio/frettaApp/pull/2/files#diff-bd250610c6b3fa979c8e0f1b71926f40L54) so if we will be logging out in more than one place in this class, we will use this single string, instead of copy-paste

"But what value does the String LOGTAG have?" --> the `getSimpleName()` method returns just the class name, which is `MyViewPagerFragment ` like we had before, but now declared in one single place for the class

"why declaring the variable as `final`" --> it is good practice to declare as much as final as you can, at least when you have class variables that can be final (it can't always be final). `final` means that you can't change to which object this variable references in the memory

"why is the name of the variable upper-case?" --> when we have final variables, we have the variable name often in upper case, and if you want to have the variable name as many words you can use `_`, e.g. `THIS_IS_VARIABLE_NAME_WITH_MANY_WORDS`,
while when you have normal variables (not final), it is often used so called "camel-case" format, e.g. `thisIsNotFinalVariable`